samples/drivers/charger/README.rst

```diff
@@ -7,4 +7,13 @@
 Overview
 ********
 
-This sample demonstrates how to use the :ref:`charger_api`.
+This sample demonstrates how to use the :ref:`charger_api` API.
+
+The sample application performs a simple charging task loop. The
+application will first poll for external power provided to the
+charger device. If power is provided to the charger, then the sample
+application will attempt to enable the charge cycle. After the
+charge cycle is initiated, the sample application will check the
+status property of the charger device and report any relevant
+information to the log. Once the charger device reports that the
+charge cycle has completed, the application returns.
```

Comment:
wrapping at 72 characters is very conservative :) 100 characters is actually the recommended limit.
Suggested reformatted version below, adding some breathing space too

```suggestion
The sample application performs a simple charging task loop.

- The application will first poll for external power provided to the charger device.
- If power is provided to the charger, then the sample application will attempt to enable the charge
cycle.
- After the charge cycle is initiated, the sample application will check the status property of the
charger device and report any relevant information to the log.
- Once the charger device reports that the charge cycle has completed, the application returns.
```